platform-test/base/unification.oz

```diff
@@ -0,0 +1,106 @@
+functor
+import
+   System(show:Show)
+export
+   Return
+define
+   ShouldHaveFailed = 'should have failed'
+   proc{MustFail X P}
+      skip
+      try
+         {P}
+         raise ShouldHaveFailed end
+      catch S then case S of !ShouldHaveFailed then
+         raise '  '(X ShouldHaveFailed) end
+      [] _ then
+         skip
+      end end
```

Comment:
Ok, I tried and got parsing exception, but the problem was in the chair-keyboard interface. ;)